packages/components/pie-cookie-banner/test/component/pie-cookie-banner.spec.ts

```diff
@@ -197,9 +161,9 @@ test.describe('PieCookieBanner - Component tests', () => {
         });
     });
 
-    test('should toggle the `all` preference node if the other preferences are set to true manually', async ({ mount }) => {
+    test('should toggle the `all` preference node if the other preferences are set to true manually', async () => {
         // Arrange
-        await mount(PieCookieBanner, { props: {} as CookieBannerProps });
+        await pieCookieBannerComponent.load({ defaultPreferences: null });
```

Comment:
The preferences should be false by default, right? 
```suggestion
        await pieCookieBannerComponent.load();
```

Also there's the `no-restricted-syntax` eslint comment below here, is that still needed? I think we may have set a rule that means that's no longer required

Reply:
Just looked and yeah, we still need that `no-restricted-syntax` line.

Reply:
Also this one does require the `defaultPreferences: null` as it's currently working backwards based on the default values. Meaning that we'll need to alter this test once we make the fixies for preferences defaults.